src/mm-grid-column/mm-grid-column.js

```diff
@@ -13,46 +13,15 @@
 			StrandTraits.Refable
 		],
 
-		properties: {
-
-			SORT_ICON_COLOR: {
-				type: String,
-				value: Colors.A2,
-			},
-			INFO_ICON_COLOR: {
-				type: String,
-				value: Colors.A4,
-			},
-
-			SORT_DEFAULT: {
-				type: Number,
-				value: 0,
-			},
-			SORT_ASCENDING: {
-				type: Number,
-				value: 1,
-			},
-			SORT_DESCENDING: {
-				type: Number,
-				value: -1,
-			},
-			SORT_EVENT: {
-				type: String,
-				value: "column-sort",
-			},
-			RESIZE_EVENT: {
-				type: String,
-				value: "column-resize",
-			},
-			RESIZE_START_EVENT: {
-				type: String,
-				value: "column-resize-start",
-			},
-			RESIZE_END_EVENT: {
-				type: String,
-				value: "column-resize-end",
-			},
+		SORT_DEFAULT: 0,
+		SORT_ASCENDING: 1,
+		SORT_DESCENDING: -1,
+		SORT_EVENT: "column-sort",
+		RESIZE_EVENT: "column-resize",
+		RESIZE_START_EVENT: "column-resize-start",
+		RESIZE_END_EVENT: "column-resize-end",
```

Comment:
so the way i did this in ajax was actually define them on scope.Ajax

so for this it would be like scope.Grid.SORT_DEFAULT

which means when the end user is using these things they could write like
`$0.sort = Strand.Grid.SORT_DEFAULT`

I dont have strong feelings though cause i know that makes a hash of the internal code, and given the likely lack of api usage to begin with i'm for making our code cleaner in this case.